bin/MCFM/cards/MCFM+JHUGen/MCFM_JHUGen_13TeV_ggZZtoMUTL_BKG.DAT

```diff
@@ -0,0 +1,142 @@
+'7.0'		[file version number]
+
+[Flags to specify the mode in which MCFM is run]
+1			[nevtrequested]
+.false.		[creatent]
+.false.		[skipnt]
+.false.		[dswhisto]
+.true.     [creategrid]
+.false.		[writetop]
+.true.		[writedat]
+.true.		[writegnu]
+.false.		[writeroot]
+.false.		[writepwg]
+
+[General options to specify the process and execution]
+132.MUTL		[nproc] 
+'lord'  	[part 'lord','real' or 'virt','tota']
+'CMS'		['runstring']
+13000d0		[sqrts in GeV]
++1		[ih1 =1 for proton and -1 for antiproton]
++1		[ih2 =1 for proton and -1 for antiproton]
+125		[hmass]
+0.5d0		[scale:QCD scale choice]
+0.5d0		[facscale:QCD fac_scale choice]
+'m(3456)'		[dynamicscale]
+.false.		[zerowidth]
+.false.		[removebr]
+4		[itmx1, number of iterations for pre-conditioning]
+100000		[ncall1]
+10		[itmx2, number of iterations for final run]
+100000		[ncall2]
+SEED		[ij]
+.false.		[dryrun]
+.true.		[Qflag]
+.true.		[Gflag]
+
+[Heavy quark masses]
+173.2d0		[top mass]
+4.75d0		[bottom mass]
+1.275d0		[charm mass]
+
+[Pdf selection]
+cteq6_l [pdlabel]
+4       [NGROUP, see PDFLIB]
+46      [NSET - see PDFLIB]
+'NNPDF31_lo_as_0131'			[LHAPDF group]
```

Comment:
would it make sense to include the pdf set (either NNPDF3 or the lhaid number) in the filename as for the powheg datacards?